src/extractor/guidance/roundabout_handler.cpp

```diff
@@ -270,13 +270,30 @@ RoundaboutType RoundaboutHandler::getRoundaboutType(const NodeID nid) const
     std::set<NodeID> roundabout_nodes; // needs to be sorted
 
     // this value is a hard abort to deal with potential self-loops
+    const auto countRoundaboutFlags = [&](const NodeID at_node) {
+        // this would be nicer as boost::count_if, but our integer range does not support these
+        // range based handlers
```

Comment:
Let's ticket the irange deficiency and where the hacks for this are in the code.
Or at least let's tag them FIXME or something.